salt/reboot/init.sls

```diff
@@ -0,0 +1,42 @@
+##################################################
+# Configuration for the reboot manager
+##################################################
+
+# `max_holders` contains the maximum number of lock holders for the cluster.
+{% set max_holders = salt['pillar.get']('etcd:masters', '1')|int %}
+{% if max_holders > 1 %}
+  {% set max_holders = max_holders - 1 %}
+{% endif %}
+
+# Cleanup any previous cluster information on /opensuse.org
+opensuseorg_cleanup:
+  pkg.installed:
+    - name: etcdctl
+  cmd.run:
+    - name: etcdctl
+            rm -r /opensuse.org
```

Comment:
wouldn't `rm -rf` be better here? i think it might fail otherwise if there was no such folder